backend/dataall/modules/datasets/cdk/dataset_custom_resources_extension.py

```diff
@@ -88,14 +88,14 @@ def extent(setup: EnvironmentSetup):
             setup,
             'LakeformationDefaultSettingsCustomeResourceFunctionArn',
             string_value=lf_default_settings_custom_resource.function_arn,
-            parameter_name=f'/dataall/{setup.environment().environmentUri}/cfn/lf/defaultsettings/lambda/arn',
```

Comment:
Minor formatting - should we jsut define `_env = setup.environment()` and then call the `_env.resourcePrefix` and `_env.environmentUri` all the subsequent times?

Reply:
Done :)